code/modules/reagents/chemistry/reagents/unique/eigenstasium.dm

```diff
@@ -0,0 +1,110 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//										EIGENSTASIUM
+///////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Don't do this, because it tricks the autodoc into thinking this is a doc comment.

Reply:
Speaking of which, make this entire comment chain a real doc comment.

Reply:
Does /* stuff */ trick it too?